### PR TITLE
Implement product list component

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement product list component

image

storybook: https://6684faab2d0c99a3e93bf145-qbdoqulsob.chromatic.com/

### CODE REVIEW

File: src/components/ProductList/ProductList.stories.tsx [hunk 1]
```diff
@@ -1,6 +1,114 @@
...
+    title: 'title 3',
+    price: 40,
+    images: [images],
+    colors: [],
+    sizes: [],
+    description: 'description 3',
+    stock: 6,
+    rate: 73,
+    numberRating: 3,
+    category: 'category 3',
```
Comment (thanhnguyen-agilityio) [inline src/components/ProductList/ProductList.stories.tsx]: Use real name of category, like `T-shirt`

File: src/components/ProductList/ProductList.stories.tsx [hunk 2]
```diff
@@ -1,6 +1,114 @@
...
+    price: 63,
+    images: [images],
+    colors: [],
+    sizes: [],
+    description: 'description 5',
+    stock: 87,
+    rate: 37,
+    numberRating: 4,
+    category: '2',
+    id: 6,
```
Comment (thanhnguyen-agilityio) [inline src/components/ProductList/ProductList.stories.tsx]: Id go first.

File: src/components/ProductList/ProductList.stories.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply src/components/ProductList/ProductList.stories.tsx]: Fixed

File: src/components/ProductList/ProductList.stories.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply src/components/ProductList/ProductList.stories.tsx]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**